json-sql

```diff
@@ -84,6 +99,24 @@ class DB:
     def _table_names(self):
         return NotImplementedError
 
+class ORACLEADB(DB):
+    schema = None
```

Comment:
It might be good style to declare class variables in python, I'm not sure, but since no other classes do this, if we remove this, we can keep the code more consistent.

Reply:
I don't use python much - just my Java developer coming through! :)